pnpm-lock.yaml

```diff
@@ -550,10 +550,10 @@ importers:
       rehype-slug: 5.1.0
       rehype-toc: 3.0.2
       remark-code-titles: 0.1.2
-      rollup: 3.14.0
-      sass: 1.58.0
+      rollup: 3.15.0
+      sass: 1.58.1
       srcset-parse: 1.1.0
-      undici: 5.18.0
+      undici: 5.19.1
```

Comment:
Looks like this may have changed some `Headers` behavior that is causing cookie tests to fail now...

Reply:
Possibly relevant? https://github.com/nodejs/undici/pull/1915